engines/agi/preagi.cpp

```diff
@@ -55,32 +56,22 @@ PreAgiEngine::PreAgiEngine(OSystem *syst, const AGIGameDescription *gameDesc) :
 
 void PreAgiEngine::initialize() {
 	initRenderMode();
+	initFont();
 
 	_gfx = new GfxMgr(this);
 	_picture = new PictureMgr(this, _gfx);
 
-	if (getGameID() == GID_MICKEY) {
-		_fontData = fontData_Mickey;
-	} else {
-		_fontData = fontData_IBM;
-	}
-
 	_gfx->initMachine();
 
 	_game.gameFlags = 0;
 
-	_game.colorFg = 15;
-	_game.colorBg = 0;
+	//_game._vm->_text->charAttrib_Set(15, 0);
```

Comment:
Because it's not needed at the moment. The preAGI code draws text by itself and doesn't use the TextMgr class for now (at least as far as I can see). I'm not sure if PreAGI should get changed accordingly or not.